lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -239,7 +239,7 @@ export default class DocumentComposer {
       // Loop through all given public keys and add them if they don't exist already.
       for (let publicKey of patch.publicKeys) {
         // NOTE: If a key ID already exists, we will just replace the existing key.
-        // Not throwing error will minimize the need (thus risk) of reusing exposed update OTP.
+        // Not throwing error will minimize the need (thus risk) of reusing exposed update commitment/reveal value.
```

Comment:
Please rename all occurrences of "commitment/reveal value" to just "reveal value".